.travis.yml

```diff
@@ -1,27 +1,20 @@
-rvm:
-  - 2.4.0
-  - 2.3.0
-  - 2.2.4
-  - 2.1.8
-  - 2.0.0
-  - jruby-19mode
-gemfile:
-  - gemfiles/rails_4.gemfile
-  - gemfiles/rails_5.0.gemfile
-  - gemfiles/rails_5.1.gemfile
-before_install:
-  - gem update --system
-  - gem update bundler
-install: "bundle install"
-
 matrix:
-  exclude:
-    # Rails 5 requires Ruby >= 2.2.2
-    - rvm: 2.0.0
+  include:
+    - rvm: 2.6
       gemfile: gemfiles/rails_5.0.gemfile
-    - rvm: 2.0.0
+    - rvm: 2.6
       gemfile: gemfiles/rails_5.1.gemfile
-    - rvm: 2.1.8
+    - rvm: 2.6
       gemfile: gemfiles/rails_5.0.gemfile
-    - rvm: 2.1.8
+    - rvm: 2.6
       gemfile: gemfiles/rails_5.1.gemfile
+    - rvm: 2.3
+      gemfile: gemfiles/rails_4.gemfile
+    - rvm: 2.6
+    - rvm: 2.5
+    - rvm: 2.4
+    - rvm: 2.3
+    - rvm: 2.2
+    - rvm: 2.1
+    - rvm: 2.0
```

Comment:
These versions have reached EOL

Reply:
I can remove them, but probably better to address removal of support separately?